config/migrations/2024/20241120124400-generic-prefix-http.sparql

```diff
@@ -0,0 +1,14 @@
+PREFIX generiekHttps: <https://data.vlaanderen.be/ns/generiek#>
+PREFIX generiek: <http://data.vlaanderen.be/ns/generiek#>
+
+INSERT {
+  GRAPH ?g {
+    ?s ?pHttp ?o.
+  }
+}WHERE {
+  GRAPH ?g {
+    ?s ?p ?o.
+  }
+  FILTER (STRSTARTS(STR(?p), STR(generiekHttps:)))
```

Comment:
let's not do this and just define all predicates manually

Reply:
the reason why:
- this is inefficient
- this way we aren't 100% sure about what we are doing